package.json

```diff
@@ -22,12 +22,13 @@
   },
   "main": "node/index.js",
   "dependencies": {
+    "babel-core": "^5",
     "babel-runtime": "^5",
     "lodash": "^3.3.1",
+    "through": "^2.3.6",
     "tiny-sprintf": "^0.3.0"
   },
   "devDependencies": {
-    "babel-core": "^5",
```

Comment:
Sure.  The gist of what I'm trying to create is a third party library that takes jsnetworkx as a dependency.  I'm needing to bundle my code up, but I'm running into some issues as npm seems to want to transform jsnetworkx through the transforms that are specified in the jsnx package.json, but aren't included.  When I un-ignored them in `.npmignore`, it appeared I needed to include some of the libraries that were in `devDependencies`.

If I take the built `jsnetworkx.js` file directly, I can build just fine but I don't get the benefit of locking down a version and such.  I'll try to put together an actual gist to illustrate what I'm trying to do.  I'm new to this ecosystem so I may very well be making some mistakes in configuration.

In any case, thanks for this library :)